src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball randomBall = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+
+        randomBall.setNumber((int) (Math.random() * 100));
+        randomBall.setColor(randomColor.getRandomColor());
+
+        return randomBall.getColor() + " " + randomBall.getNumber();
+    }
+}
```

Comment:
It seems you're using the `ColorSupplier` class directly in the `getRandomBall` method. According to the checklist, you should avoid creating several instances of the same class if you can use only one instance for your purposes. Consider creating a single instance of `ColorSupplier` at the class level and reusing it.